B1

okannako opened the pull request:
Node Name : aokannn2
Node Id : 24d92aac7617a140
Link : https://dashboard.testnet.concordium.com/node/24d92aac7617a140

[1 - concordium-testnet-node-b96d596a1623f5a47e8ab98333e61aa2926d9f85d6fc702e282f38db8da07671.log](https://github.com/Concordium/Testnet3-Challenges/files/5580688/1.-.concordium-testnet-node-b96d596a1623f5a47e8ab98333e61aa2926d9f85d6fc702e282f38db8da07671.log)
[2 - concordium-testnet-node-a8c7a2788f5d8ea0f0089bdbda89ccf9d9d49401a3f19d247e6ad5b35a3d12d4.log](https://github.com/Concordium/Testnet3-Challenges/files/5580689/2.-.concordium-testnet-node-a8c7a2788f5d8ea0f0089bdbda89ccf9d9d49401a3f19d247e6ad5b35a3d12d4.log)
[concordium-testnet-system-report.log](https://github.com/Concordium/Testnet3-Challenges/files/5580690/concordium-testnet-system-report.log)

okannako committed:
Add files via upload

1-) 43PprU3PVSdNoNjWREFE6jE2tB6GubuQHPWCL7Z7tqgWB3s3E1
2-) 4kyy1PEm8F9AFsjbaqDneNn2uFQ4hrhwYCg5o9iUTFg3YatsUf

Android Oreo 8.0

okannako committed:
Add files via upload

3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
Android Oreo 8.0, Pc

okannako committed:
Add files via upload

Baker ID - 1177
Account address - 3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw

okannako committed:
B4

Baker ID: 1198
Adres: 33BHt7wbVN4DVLGzAvHMp4QY5JZi9KtiUBgAfx1y5qUtyQQwFu

B4

okannako committed:
B3

Node ID: b0730baed1ee2ecf
okkan4
B3

Reach time after 1st stop and normal start: 32,03 seconds

Reach time after 2st stop and ./concordium-node --no-block-state-import start: 23,48 seconds

okannako committed:
B4

Baker ID - 1177
Account address - 3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw

okannako committed:
B5

Baker ID: 1198
Adres: 33BHt7wbVN4DVLGzAvHMp4QY5JZi9KtiUBgAfx1y5qUtyQQwFu

B5

okannako committed:
B3 - 2

Node ID: 367cbede61be34bf
Node Name: kara00
B3

Reach time after 1st stop and normal start: 29,32

Reach time after 2st stop and ./concordium-node --no-block-state-import start: 24,52

okannako committed:
T1

A Account: 3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
B Account: 399E63J1EcgR5T27WFoHjBjfJALzoy1tZd2MT8mcTK78ysdH21

Android Oreo Pc - Windows 10

okannako committed:
T2

A: okan - 3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
B: okan2 - 399E63J1EcgR5T27WFoHjBjfJALzoy1tZd2MT8mcTK78ysdH21

T2

okannako committed:
T2 

A: okan - 3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
B: okan2 - 399E63J1EcgR5T27WFoHjBjfJALzoy1tZd2MT8mcTK78ysdH21

T2

okannako committed:
T3

Account 1 - okan : 3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
Account 2 okan2 : 399E63J1EcgR5T27WFoHjBjfJALzoy1tZd2MT8mcTK78ysdH21

Time : [19:23:12] to [23:57:04] Turkey

okannako committed:
B6 

Name: aokannn2
Id: 24d92aac7617a140

okannako committed:
B2

Name: kara3
Id: fdc39c4cea2ff081

okannako committed:
B6 - 2

Name: kara3
Id: fdc39c4cea2ff081

okannako committed:
Merge pull request #1 from okannako/okannako-patch-1

T2

okannako committed:
B3

Node ID: 367cbede61be34bf
Node Name: kara00
B3

Reach time after 1st stop and normal start: 29,32

Reach time after 2st stop and ./concordium-node --no-block-state-import start: 24,52

okannako committed:
B2 - Update

Node ID: 367cbede61be34bf
Node Name: kara00
B3

Reach time after 1st stop and normal start: 29,32

Reach time after 2st stop and ./concordium-node --no-block-state-import start: 24,52

okannako committed:
B3 - Update

Node ID: 367cbede61be34bf
Node Name: kara00
B3

Reach time after 1st stop and normal start: 29,32

Reach time after 2st stop and ./concordium-node --no-block-state-import start: 24,52

okannako committed:
B3 - Update

Node ID: b0730baed1ee2ecf
okkan4
B3

Reach time after 1st stop and normal start: 32,03 seconds

Reach time after 2st stop and ./concordium-node --no-block-state-import start: 23,48 seconds

okannako committed:
EI6

Name: aokannn2
Id: 24d92aac7617a140

Android Oreo 8.1 PC - Windows 10

okannako committed:
EI4

Android Oreo 8.1 for PC - Windows 10

okannako committed:
B1

Node Name : aokannn2
Node Id : 24d92aac7617a140
Link : https://dashboard.testnet.concordium.com/node/24d92aac7617a140

supermario commented:
confirmed:EI4

cryptovoyagr commented:
pending:ID1
account1:screenshot-TBC
account2:screenshot-TBC

We cannot fully verify this submission. Please assist us in typing out the account addresses for ID1 in full as text in a comment on this PR. Thank you!
@okannako

okannako commented:
> pending:ID1
> account1:screenshot-TBC
> account2:screenshot-TBC
> 
> We cannot fully verify this submission. Please assist us in typing out the account addresses for ID1 in full as text in a comment on this PR. Thank you!
> @okannako

1-) 43PprU3PVSdNoNjWREFE6jE2tB6GubuQHPWCL7Z7tqgWB3s3E1
2-) 4kyy1PEm8F9AFsjbaqDneNn2uFQ4hrhwYCg5o9iUTFg3YatsUf

Android Oreo 8.0

![111](https://user-images.githubusercontent.com/73176377/103405363-e1fb7580-4b67-11eb-905f-785e5d289152.png)
![1111](https://user-images.githubusercontent.com/73176377/103405368-e7f15680-4b67-11eb-99b0-b0ff45a0bd4e.png)

@cryptovoyagr

supermario commented:
confirmed:ID1
account1:43PprU3PVSdNoNjWREFE6jE2tB6GubuQHPWCL7Z7tqgWB3s3E1
account2:4kyy1PEm8F9AFsjbaqDneNn2uFQ4hrhwYCg5o9iUTFg3YatsUf

supermario commented:
confirmed:B1
node:24d92aac7617a140

supermario commented:
confirmed:EI6

supermario commented:
confirmed:B2

supermario commented:
confirmed:B6

supermario commented:
rejected:B3
reason:missing logs for 3 different startup types

okannako commented:
> reddedildi: B3
> nedeni: 3 farklı başlangıç ​​türü için eksik günlükler


Why did you offer such a reason for B3, would you check the logs again ?
[b3.zip](https://github.com/Concordium/Testnet3-Challenges/files/5789717/b3.zip)
[info.txt](https://github.com/Concordium/Testnet3-Challenges/files/5789718/info.txt)
@supermario

supermario commented:
confirmed:B3

cryptovoyagr commented:
pending:T1
account:screenshot-TBC
DeviceDetails:screenshot-TBC
reason:We cannot fully verify this submission. Please assist us in typing out the account addresses for T1 in full as text in a comment on this PR. Thank you!

@okannako

okannako commented:
> pending:T1
> account:screenshot-TBC
> DeviceDetails:screenshot-TBC
> reason:We cannot fully verify this submission. Please assist us in typing out the account addresses for T1 in full as text in a comment on this PR. Thank you!
> 
> @okannako

A Account: 3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
B Account: 399E63J1EcgR5T27WFoHjBjfJALzoy1tZd2MT8mcTK78ysdH21

Android Oreo Pc - Windows 10

![7 (1)](https://user-images.githubusercontent.com/73176377/104325000-3227ef80-54f9-11eb-92b3-fd3aca132513.png)
![7 (2)](https://user-images.githubusercontent.com/73176377/104325011-33591c80-54f9-11eb-8098-c3f3e3ddf683.png)
![7 (3)](https://user-images.githubusercontent.com/73176377/104325017-348a4980-54f9-11eb-886e-cca17d3a067d.png)
![7 (4)](https://user-images.githubusercontent.com/73176377/104325020-3522e000-54f9-11eb-9947-2a3e68a008c6.png)
![7 (5)](https://user-images.githubusercontent.com/73176377/104325029-37853a00-54f9-11eb-9efd-d90d433a80eb.png)
![7 (6)](https://user-images.githubusercontent.com/73176377/104325030-381dd080-54f9-11eb-886b-b3c1f656f689.png)
![10-1](https://user-images.githubusercontent.com/73176377/104325034-38b66700-54f9-11eb-8170-c42cceb9fc63.png)
![10-2](https://user-images.githubusercontent.com/73176377/104325042-3b18c100-54f9-11eb-9feb-ddb734efb50a.png)
![A - B](https://user-images.githubusercontent.com/73176377/104325044-3bb15780-54f9-11eb-9ce3-00cc70aa8a4e.png)

thank you.

@cryptovoyagr

cryptovoyagr commented:
confirmed:ID4
AccountAddress:3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
DeviceDetails:verified

cryptovoyagr commented:
confirmed:T1
account:3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
DeviceDetails:399E63J1EcgR5T27WFoHjBjfJALzoy1tZd2MT8mcTK78ysdH21

cryptovoyagr commented:
pending:T2
account:screenshot-TBC
reason:We cannot fully verify this submission. Please assist us in typing out the account addresses for T2 in full as text in a comment on this PR. Thank you!
@okannako

okannako commented:
> pending:T2
> account:screenshot-TBC
> reason:We cannot fully verify this submission. Please assist us in typing out the account addresses for T2 in full as text in a comment on this PR. Thank you!
> @okannako

![A - A 10 1](https://user-images.githubusercontent.com/73176377/104748770-25a6cf80-5763-11eb-8f99-dca3dadf3177.JPG)
![A - A 10 2](https://user-images.githubusercontent.com/73176377/104748772-263f6600-5763-11eb-9a47-0f83c8bc22fa.JPG)
![A - A 10 3](https://user-images.githubusercontent.com/73176377/104748775-263f6600-5763-11eb-81bc-c1e1cfb0a7e2.JPG)
![A - A 10 4](https://user-images.githubusercontent.com/73176377/104748777-26d7fc80-5763-11eb-819a-7cfd63831c8f.png)
![A - A 10 5](https://user-images.githubusercontent.com/73176377/104748780-26d7fc80-5763-11eb-9fe2-6d08c92701bd.png)
![A - A 10 6](https://user-images.githubusercontent.com/73176377/104748784-27709300-5763-11eb-9640-8c40867c3904.png)
![A - A 10 7](https://user-images.githubusercontent.com/73176377/104748787-27709300-5763-11eb-9f32-2b5f665c726d.png)
![A - B 3 - 1](https://user-images.githubusercontent.com/73176377/104748789-28092980-5763-11eb-85e9-e958e0817072.JPG)
![A - B 3 - 2](https://user-images.githubusercontent.com/73176377/104748792-28092980-5763-11eb-8632-512449e074fe.JPG)
![A - B 3 - 3](https://user-images.githubusercontent.com/73176377/104748793-28a1c000-5763-11eb-9682-85164fd8187f.JPG)
![A - B 3 - 4](https://user-images.githubusercontent.com/73176377/104748795-293a5680-5763-11eb-938f-6dd0c2ebdb11.png)
![A - B 3 - 5](https://user-images.githubusercontent.com/73176377/104748796-293a5680-5763-11eb-9c46-ed476cf5545c.png)
![A - B 3 - 6](https://user-images.githubusercontent.com/73176377/104748797-29d2ed00-5763-11eb-87e8-c2f493c6822c.png)
![A - B 3 - 7](https://user-images.githubusercontent.com/73176377/104748800-29d2ed00-5763-11eb-8fb6-9671f8289e98.png)
![A B](https://user-images.githubusercontent.com/73176377/104748801-29d2ed00-5763-11eb-8e71-5585c3780bdc.png)
![A1](https://user-images.githubusercontent.com/73176377/104748802-2a6b8380-5763-11eb-9b80-bb0741943014.png)
![A2](https://user-images.githubusercontent.com/73176377/104748805-2a6b8380-5763-11eb-9dd2-171cc4265ec8.png)
![B1](https://user-images.githubusercontent.com/73176377/104748807-2b041a00-5763-11eb-89d9-676be7c05ed3.png)
![B2](https://user-images.githubusercontent.com/73176377/104748808-2b041a00-5763-11eb-8b4b-0ca8499daaa3.png)
![Se A- B - 7 - 1](https://user-images.githubusercontent.com/73176377/104748811-2b9cb080-5763-11eb-8c16-d1c998aec0b3.JPG)
![Se A- B - 7 - 2](https://user-images.githubusercontent.com/73176377/104748812-2b9cb080-5763-11eb-8bed-32cf853ef729.JPG)
![Se A- B - 7 - 3](https://user-images.githubusercontent.com/73176377/104748813-2c354700-5763-11eb-848f-7643f9068d05.JPG)
![Se A- B - 7 - 4](https://user-images.githubusercontent.com/73176377/104748815-2c354700-5763-11eb-85a2-bce8a270d62f.png)
![Se A- B - 7 - 5](https://user-images.githubusercontent.com/73176377/104748817-2c354700-5763-11eb-9df3-843d664049df.png)
![Se A- B - 7 - 6](https://user-images.githubusercontent.com/73176377/104748820-2ccddd80-5763-11eb-9afa-50035cf54225.png)
![Se A- B - 7 - 7](https://user-images.githubusercontent.com/73176377/104748821-2ccddd80-5763-11eb-9af3-cec5c745af36.png)
![Se B - B - 7 - 1](https://user-images.githubusercontent.com/73176377/104748824-2d667400-5763-11eb-8fbb-9f67f7effa40.JPG)
![Se B - B - 7 - 2](https://user-images.githubusercontent.com/73176377/104748828-2d667400-5763-11eb-8a1c-dc22a85e91e9.JPG)
![Se B - B - 7 - 3](https://user-images.githubusercontent.com/73176377/104748830-2dff0a80-5763-11eb-8e00-86f4ef4d9a87.JPG)
![Se B - B - 7 - 4](https://user-images.githubusercontent.com/73176377/104748832-2dff0a80-5763-11eb-90d6-d690498dc798.png)
![Se B - B - 7 - 5](https://user-images.githubusercontent.com/73176377/104748834-2e97a100-5763-11eb-9c67-be8f5f531541.png)
![Se B - B - 7 - 6](https://user-images.githubusercontent.com/73176377/104748836-2e97a100-5763-11eb-9d6d-99ea24f6d758.png)
![Se B - B - 7 - 7](https://user-images.githubusercontent.com/73176377/104748838-2f303780-5763-11eb-9010-9c4f5638c421.png)

Thank You
@cryptovoyagr

cryptovoyagr commented:
confirmed:T3
address:3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw

okannako commented:
> confirmed:T3
> address:3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw

hi 
I thought I installed **T4** but it obviously doesn't look like a mistake, but I did the task. My mission information is below, and when you look, you will see that I did it on time thank you.

[Accounts.txt](https://github.com/Concordium/Testnet3-Challenges/files/5826131/Accounts.txt)
[concordium-testnet-node-6473c369287f542c22d8fb20267debab0b99ed4ad3cc0b583da3b099938df10c.log](https://github.com/Concordium/Testnet3-Challenges/files/5826132/concordium-testnet-node-6473c369287f542c22d8fb20267debab0b99ed4ad3cc0b583da3b099938df10c.log)
![okan](https://user-images.githubusercontent.com/73176377/104844067-f74dff00-58de-11eb-8a32-0ce8228b0908.png)
![okan2](https://user-images.githubusercontent.com/73176377/104844069-fa48ef80-58de-11eb-8ea7-76d0915c0b8b.png)

@cryptovoyagr

cryptovoyagr commented:
confirmed:T1
account1:3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
account2:399E63J1EcgR5T27WFoHjBjfJALzoy1tZd2MT8mcTK78ysdH21
DeviceDetails:verified

okannako committed:
T4

Name: kara3
Id: fdc39c4cea2ff081

okan : 3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
okan2: 399E63J1EcgR5T27WFoHjBjfJALzoy1tZd2MT8mcTK78ysdH21

supermario commented:
confirmed:B4
address:3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
baker:1048

cryptovoyagr commented:
confirmed:T2
account1:3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
account2:399E63J1EcgR5T27WFoHjBjfJALzoy1tZd2MT8mcTK78ysdH21

okannako commented:
> confirmed:T2
> account1:3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
> account2:399E63J1EcgR5T27WFoHjBjfJALzoy1tZd2MT8mcTK78ysdH21

Hello;
B5 and T4 my tasks are not checked, thank you
@cryptovoyagr @supermario

supermario commented:
rejected:B5
address:3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
reason: no 'update baker account' or 'update baker key' transactions found on-chain

okannako commented:
> rejected:B5
> address:3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
> reason: no 'update baker account' or 'update baker key' transactions found on-chain

Baker ID: 1198
Adres: 33BHt7wbVN4DVLGzAvHMp4QY5JZi9KtiUBgAfx1y5qUtyQQwFu

B5

![1](https://user-images.githubusercontent.com/73176377/106631193-92a4cc80-658d-11eb-86f7-9d7de5fc5ae0.JPG)
![2](https://user-images.githubusercontent.com/73176377/106631209-959fbd00-658d-11eb-9b5e-70a3399d04aa.JPG)
![3](https://user-images.githubusercontent.com/73176377/106631211-96385380-658d-11eb-955d-d26b1f8f3c5c.JPG)
![4](https://user-images.githubusercontent.com/73176377/106631214-96385380-658d-11eb-9b92-0ce991de554f.JPG)
![5](https://user-images.githubusercontent.com/73176377/106631216-96d0ea00-658d-11eb-8518-0adedd26efb0.JPG)
![6](https://user-images.githubusercontent.com/73176377/106631218-96d0ea00-658d-11eb-8405-00ab7b7bce5e.JPG)
![7](https://user-images.githubusercontent.com/73176377/106631219-97698080-658d-11eb-9f0c-a0d6ab00ce64.png)
![8](https://user-images.githubusercontent.com/73176377/106631222-97698080-658d-11eb-8311-9ecc9badd862.png)
![9](https://user-images.githubusercontent.com/73176377/106631225-98021700-658d-11eb-8ce5-09cc24f59c55.png)
![10](https://user-images.githubusercontent.com/73176377/106631228-98021700-658d-11eb-9f7a-07b054e0b370.png)
![11](https://user-images.githubusercontent.com/73176377/106631231-98021700-658d-11eb-8bc0-f7eaf8f0df35.png)

@supermario

supermario commented:
rejected:B5
address:33BHt7wbVN4DVLGzAvHMp4QY5JZi9KtiUBgAfx1y5qUtyQQwFu
reason: no 'update baker account' or 'update baker key' transactions found on-chain

Hey @okannako  – I did check this again with the new address you gave, but unfortunately the [B5 requirements](https://github.com/Concordium/Testnet3-Challenges/pull/1728) don't seem to be met:

- update baker's reward account in the Concordium Client
- update baker's keys in the Concordium Client

okannako commented:
> rejected:B5
> address:33BHt7wbVN4DVLGzAvHMp4QY5JZi9KtiUBgAfx1y5qUtyQQwFu
> reason: no 'update baker account' or 'update baker key' transactions found on-chain
> 
> Hey @okannako – I did check this again with the new address you gave, but unfortunately the [B5 requirements](https://github.com/Concordium/Testnet3-Challenges/pull/1728) don't seem to be met:
> 
> * update baker's reward account in the Concordium Client
> * update baker's keys in the Concordium Client

 I remember doing the job, but there's a mistake somewhere. I have these accounts, if you can look at them, they can be, but I don't have baker key. If it doesn't happen that way, this task will be incomplete. :)
 
 1: 3W4EbUdvmDFizYeRFCDQ2gdCN2WdKzh5dzdgHtHJpeeEwpCxDw
 2: 399E63J1EcgR5T27WFoHjBjfJALzoy1tZd2MT8mcTK78ysdH21

okannako commented:
Form Submitted: 03:39 AM UTC+3
